scripts/hrrr_downloader.py

```diff
@@ -58,18 +77,41 @@ def setupArgs() -> None:
   )
   parser.add_argument(
     "--outputDir",
-    default="data/weather_data/",
+    default="../../../data0/balaji24/data/weather_data/",
```

Comment:
```suggestion
    default="data/weather_data/",
```

(i'd like to keep the default generic)

Reply:
Changed this. Thank you!